dev-env-havener.yaml

```diff
@@ -29,16 +29,22 @@
 
 name: cf-operator-quarks
 releases:
-- chart_name: cf-operator
-  chart_namespace: quarks
-  chart_location: deploy/helm/cf-operator/
+- name: cf-operator
+  namespace: quarks
+  location: deploy/helm/cf-operator/
   overrides:
     image:
-      tag: (( shell . bin/include/versioning && echo "${VERSION_TAG}" )) 
-      repository: cf-operator
-      org: cfcontainerization
+      tag: (( env IMAGE_TAG ))
```

Comment:
we already have `DOCKER_IMAGE_TAG`, can we use that here too?

Reply:
yes